openoa/analysis/aep.py

```diff
@@ -74,21 +78,21 @@ class MonteCarloAEP(object):
     @logged_method_call
     def __init__(
         self,
-        plant,
-        reanal_products=["merra2", "ncep2", "erai", "era5"],
-        uncertainty_meter=0.005,
-        uncertainty_losses=0.05,
-        uncertainty_windiness=(10, 20),
-        uncertainty_loss_max=(10, 20),
-        outlier_detection=False,
-        uncertainty_outlier=(1, 3),
-        uncertainty_nan_energy=0.01,
-        time_resolution="M",
-        end_date_lt=None,
-        reg_model="lin",
-        ml_setup_kwargs={},
-        reg_temperature=False,
-        reg_winddirection=False,
+        plant: PlantData,
+        reanal_products: list[str] = ["merra2", "ncep2", "erai", "era5"],
+        uncertainty_meter: float = 0.005,
+        uncertainty_losses: float = 0.05,
+        uncertainty_windiness: tuple[int] = (10, 20),
```

Comment:
Just for reference, tuples have to be annotated as either `tuple[int, int]` in this case, or for a dynamic number of values: `tuple[int, ...]`.